lib/middleware.js

```diff
@@ -191,8 +192,37 @@ module.exports = function(srf, logger) {
           const sipRealm = arr[2];
           logger.debug(`looking for device calling app for realm ${sipRealm}`);
           app = await lookupAppByRealm(sipRealm);
-          if (app) logger.debug({app}, `retrieved device calling app for realm ${sipRealm}`);
-
+          if (app) {
+            logger.debug({app}, `retrieved device calling app for realm ${sipRealm}`);
+          } else {
+            const calledAor = `${req.calledNumber}@${sipRealm}`;
+            const reg = await registrar.query(calledAor);
+            if (reg) {
+              logger.debug(`There is no device app found, called client is registered,
+ forwarding call to called client.`);
+              app = {
+                // Dummy hook to follow later feature server logic.
+                call_hook: {
```

Comment:
I guess something is required here?  It will never get used though right?

Reply:
right, the rest of feature server logic need this info to initiate data, but never be used in p2p call.